spec/README.md

```diff
@@ -186,6 +186,37 @@ Field Name | Type | Description
 <a name="parametersDefault"></a>default | `string` | **Optional** An optional default value for the parameter.
 <a name="parametersHidden"></a>hidden | `string` | **Optional** An optional boolean signifying the parameter should be obscured when displayed.
 
+##### XPathing
+XPathing is supported by the Nulecule spec as by [RFC 6902](https://tools.ietf.org/html/rfc6902).
```

Comment:
its "Nulecule Specification", and I can see just one reference to XPath in that RFC, any better source of information?